### PR TITLE
Added support for Export to keras retinanet CSV

### PR DESCRIPTION
**Pull request by aribornstein**
https://github.com/fizyr/keras-retinanet

### CODE REVIEW

**Comment by aribornstein** [comment]
@PIC123  let me know when you guys can review this i've tested it locally but i'd like a review for code quality purposes

**Comment by aribornstein** [comment]
Can someone review this?

**Comment by wbreza** [comment]
@aribornstein - Do you want to take a stab re-implementing in v2?

**Comment by JacopoMangiavacchi** [comment]
Hey @aribornstein, if you want to take this to v2 and eventually have any questions about v2 exporter just let me know.

File: src/lib/detection_algorithms/keras_retinanet/exporter.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+// The Exporter interface - provides a mean to export the tagged frames
+// data in the expected data format of the detection algorithm
+// Constructor parameters:
+//  exportDirPath - path to the directory where the exported file will be placed
+//  classes - list of classes supported by the tagged data
+//  posFramesCount - number of positive tagged frames
+//  frameWidth - The width (in pixels) of the image frame
+//  frameHeight - The height (in pixels) of the image frame
+//  testSplit - the percent of tragged frames to reserve for test set defaults to 20%
+function Exporter(exportDirPath, classes, posFramesCount, frameWidth, frameHeight, testSplit) {
```
Comment (wbreza) [inline src/lib/detection_algorithms/keras_retinanet/exporter.js]: Code looks good - but we should write some unit tests for this new exporter.

**Pull request closed without merging**